src/containers/Tenant/Diagnostics/Describe/Describe.tsx

```diff
@@ -0,0 +1,86 @@
+import {useEffect} from 'react';
+import cn from 'bem-cn-lite';
+import {useDispatch, useSelector} from 'react-redux';
+import {Loader} from '@gravity-ui/uikit';
+import 'react-json-inspector/json-inspector.css';
+// @ts-ignore
+import JSONTree from 'react-json-inspector';
+
+import {getDescribe} from '../../../../store/reducers/describe';
+import {useAutofetcher} from '../../../../utils/hooks';
+
+import './Describe.scss';
+
+const b = cn('kv-describe');
+
+const expandMap = new Map();
+
+interface IDescribeProps {
+    tenant: string;
+}
+
+const Describe = ({tenant}: IDescribeProps) => {
+    const dispatch = useDispatch();
+
+    const {
+        currentDescribe: data,
+        error,
+        loading,
+        wasLoaded,
+    } = useSelector((state: any) => state.describe);
+    const {autorefresh, currentSchemaPath} = useSelector((state: any) => state.schema);
+
+    const fetchData = () => {
+        const path = currentSchemaPath || tenant;
+        dispatch(getDescribe({path}));
+    };
+
+    useAutofetcher(fetchData, [], autorefresh);
```

Comment:
why empty deps array?

Reply:
It seems like I broke some of the commits with fixups. Or, the opposite, I fixed my mistakes by next commits, but not fixups. And it spoils reviewing by commits. I will try to do it more carefully further